queries.yaml

```diff
@@ -335,6 +335,60 @@ post_process:
       start_zoom: 4
       properties: [barrier]
 
+  # coarsen the landuse kinds at mid & low zooms to help improve merging.
+  #
+  # do this _early_; before we assign sort_rank and to avoid inconsistency when
+  # we join (e.g: roads) onto landuse and wouldn't want the road landuse_kind to
+  # be different from the polygon in case that makes it the wrong colour.
+  #
+  # **NOTE** WE'RE CHANGING THE KINDS HERE!
+  - fn: vectordatasource.transform.remap
+    params:
+      layer: landuse
+      start_zoom: 0
```

Comment:
Generally more consistent to start at zoom 0+, but we've modified some other configs to be 9+ only. Should we do the same here?

(We show `national_park` earlier (not covered here), but I think most everything else – especially this type of landcover remapping – starts at zoom 9.)

Reply:
The zoom ranges in the YAML files allow `residential` down to zoom 4 and `natural_wood` / `wood` down to 6, and a bunch of others down to 8. Whether any are actually that large is questionable, and I'm running a query to find out - although it may take some time to run.

We start bringing in OSM landuse starting at z4+, so we could change it to that, but at the moment any higher `start_zoom` would potentially mean stuff gets "un-generalised" when zooming out. Also, we've had a problem in the past with lack of coordination between `start_zoom` when we've changed the zoom that we bring stuff in at, or start simplifying, so I'm in two minds about whether to be "minimal" and cover only the zoom ranges at which we have data, or "defensive against bugs" by covering the whole zoom range down to zero.

Reply:
I'm fine leaving it 0.